Samples/OpenQasmReader.Tests/TestFunctions/qft_n30.qasm

```diff
@@ -0,0 +1,2239 @@
+cx q[29],q[7];
+u1(3.74507028292393e-7) q[7];
+u1(7.49014056584786e-7) q[29];
+cx q[29],q[8];
+u1(-7.49014056584786e-7) q[8];
+cx q[29],q[8];
+u1(7.49014056584786e-7) q[8];
+u1(1.49802811316957e-6) q[29];
+cx q[29],q[9];
+u1(-1.49802811316957e-6) q[9];
+cx q[29],q[9];
+u1(1.49802811316957e-6) q[9];
+u1(2.99605622633914e-6) q[29];
+cx q[29],q[10];
+u1(-2.99605622633914e-6) q[10];
+cx q[29],q[10];
+u1(2.99605622633914e-6) q[10];
+u1(5.99211245267829e-6) q[29];
+cx q[29],q[11];
+u1(-5.99211245267829e-6) q[11];
+cx q[29],q[11];
+u1(5.99211245267829e-6) q[11];
+u1(1.19842249053566e-5) q[29];
+cx q[29],q[12];
+u1(-1.19842249053566e-5) q[12];
+cx q[29],q[12];
+u1(1.19842249053566e-5) q[12];
+u1(2.39684498107131e-5) q[29];
+cx q[29],q[13];
+u1(-2.39684498107131e-5) q[13];
+cx q[29],q[13];
+u1(2.39684498107131e-5) q[13];
+u1(4.79368996214263e-5) q[29];
+cx q[29],q[14];
+u1(-4.79368996214263e-5) q[14];
+cx q[29],q[14];
+u1(4.79368996214263e-5) q[14];
+u1(9.58737992428526e-5) q[29];
+cx q[29],q[15];
+u1(-9.58737992428526e-5) q[15];
+cx q[29],q[15];
+u1(9.58737992428526e-5) q[15];
+u1(0.000191747598485705) q[29];
+cx q[29],q[16];
+u1(-0.000191747598485705) q[16];
+cx q[29],q[16];
+u1(0.000191747598485705) q[16];
+u1(0.000383495196971410) q[29];
+cx q[29],q[17];
+u1(-0.000383495196971410) q[17];
+cx q[29],q[17];
+u1(0.000383495196971410) q[17];
+u1(0.000766990393942821) q[29];
+cx q[29],q[18];
+u1(-0.000766990393942821) q[18];
+cx q[29],q[18];
+u1(0.000766990393942821) q[18];
+u1(0.00153398078788564) q[29];
+cx q[29],q[19];
+u1(-0.00153398078788564) q[19];
+cx q[29],q[19];
+u1(0.00153398078788564) q[19];
+u1(0.00306796157577128) q[29];
+cx q[29],q[20];
+u1(-0.00306796157577128) q[20];
+cx q[29],q[20];
+u1(0.00306796157577128) q[20];
+u1(0.00613592315154256) q[29];
+cx q[29],q[21];
+u1(-0.00613592315154256) q[21];
+cx q[29],q[21];
+u1(0.00613592315154256) q[21];
+u1(0.0122718463030851) q[29];
+cx q[29],q[22];
+u1(-0.0122718463030851) q[22];
+cx q[29],q[22];
+u1(0.0122718463030851) q[22];
+u1(0.0245436926061703) q[29];
+cx q[29],q[23];
+u1(-0.0245436926061703) q[23];
+cx q[29],q[23];
+u1(0.0245436926061703) q[23];
+u1(0.0490873852123405) q[29];
+cx q[29],q[24];
+u1(-0.0490873852123405) q[24];
+cx q[29],q[24];
+u1(0.0490873852123405) q[24];
+u1(0.0981747704246810) q[29];
+cx q[29],q[25];
+u1(-0.0981747704246810) q[25];
+cx q[29],q[25];
+u1(0.0981747704246810) q[25];
+u1(0.196349540849362) q[29];
+cx q[29],q[26];
+u1(-0.196349540849362) q[26];
+cx q[29],q[26];
+u1(0.196349540849362) q[26];
+u1(0.392699081698724) q[29];
+cx q[29],q[27];
+u1(-0.392699081698724) q[27];
+cx q[29],q[27];
+u1(0.392699081698724) q[27];
+u1(0.785398163397448) q[29];
+cx q[29],q[28];
+u1(-0.785398163397448) q[28];
+cx q[29],q[28];
+u1(0.785398163397448) q[28];
+h q[29];
+barrier q[0],q[1],q[2],q[3],q[4],q[5],q[6],q[7],q[8],q[9],q[10],q[11],q[12],q[13],q[14],q[15],q[16],q[17],q[18],q[19],q[20],q[21],q[22],q[23],q[24],q[25],q[26],q[27],q[28],q[29];
+measure q[0] -> c[0];
```

Comment:
any idea of why do these measurements do not generate any Q# code?

Reply:
Wow... how did I miss that one. Seems part of the barrier blocking the measure in this case.